src/inputs/input-transformers.js

```diff
@@ -0,0 +1,33 @@
+import _ from 'underscore';
+
+let normalizeItemsOption = (input) => {
+    if (!input.options.items) {
```

Comment:
should this also check for input type? What if someone adds an `-items` to a text input. Feels like when we finally get this functionality into juttle/juttle it would be up to each input type to perform its own "transformers".